generators/deployment/index.js

```diff
@@ -69,6 +69,9 @@ module.exports = class extends Generator {
 			this.manifestConfig.name = this.bluemix.name;
 			this.deployment.type = this.bluemix.cloudDeploymentType || 'CF';
 		}
+		if (this.manifestConfig.domain && this.manifestConfig.host) {
+			this.manifestConfig.route = `${this.bluemix.server.domain}/${this.bluemix.server.host}`
```

Comment:
The current syntax of the Bluemix routes are `hostname.domain`.  Is that changing to this new format?  For example, my node app I pushed earlier today has: hostname: **get-started-node-smiling-kangaroo** and domain: **mybluemix.net**  to produce its route **get-started-node-smiling-kangaroo.mybluemix.net**.